client/src/types/statistic.ts

```diff
@@ -0,0 +1,19 @@
+/**
+  This file contains the types for trap speed payloads 
+*/
+
+import {
+  Number,
+  Record,
+  Static,
+} from 'runtypes';
+
+export const PredictedPayload = Record({
+  speed: Number,
+});
+export type PredictedPayload= Static<typeof PredictedPayload>
+
+export const AchievedPayload = Record({
+  speed: Number,
+});
+export type AchievedPayload = Static<typeof AchievedPayload>
```

Comment:
Both `PredictedPayload` and `AchievedPayload` actually have the same type - they're both a dictionary with a 'speed' key. So we could probably just use one and name it generically, like speedPayload, and use this everywhere we currently use `AchievedPayload` and `PredictedPayload`. Thoughts?

Reply:
well ye that is so true !!